src/main/java/core/web/filters/AuthenticationFilter.java

```diff
@@ -0,0 +1,50 @@
+package core.web.filters;
+
+import core.lib.Injector;
+import core.service.driver.DriverService;
+import java.io.IOException;
+import javax.servlet.Filter;
+import javax.servlet.FilterChain;
+import javax.servlet.FilterConfig;
+import javax.servlet.ServletException;
+import javax.servlet.ServletRequest;
+import javax.servlet.ServletResponse;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+public class AuthenticationFilter implements Filter {
+    private static final Injector injector
+            = Injector.getInstance("core");
+    private static final String DRIVER_ID = "driverId";
+    private final DriverService driverService
+            = (DriverService) injector.getInstance(DriverService.class);
+    
+    @Override
+    public void init(FilterConfig filterConfig) throws ServletException {
+        throw new RuntimeException("Not supported");
```

Comment:
Why you throw an exception in `init()` and `destroy()`?

Reply:
You are right. Thanks